articles/other/other-how-delete-sso-user.md

```diff
@@ -0,0 +1,190 @@
+---
+title: How to delete single sign on users | UKCloud Ltd
+description: Describes the process for deleting single sign on (SSO) users
+services: other
+author: Sue Highmoor
+
+toc_rootlink: How To
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Delete SSO users
+toc_fullpath: How To/other-how-delete-sso-user.md
+toc_mdlink: other-how-delete-sso-user.md
+---
+
+# How to delete single sign on users
+
+## Overview
+
+When you use the UKCloud Portal to remove a user from an account that includes single sign on (SSO) enabled services, it's important to note that this does not remove the corresponding SSO user. To remove the SSO user, you must use the Identity & Access Management (IDAM) API.
```

Comment:
It might be better to add the abbreviation to the article title and rephrase this as 'SSO-enabled services' as it is on the next line. The lack of a hyphen before 'enabled' is inconsistent.

Reply:
Fixed